cosmo_tester/test_suites/snapshots/__init__.py

```diff
@@ -55,6 +56,33 @@
     TENANT_DEPLOYMENTS_PATH + '/{name}'
 )
 
+MANAGER_VERSIONS_LIST = (
```

Comment:
Semantically it is a list of manager versions.
Pedantically, it is a tuple, but does contain a (non-python sense) list of manager versions. <smiles>